templates/store/snap-distro-install.html

```diff
@@ -44,159 +44,150 @@
 {% endblock %}
 
 {% block content %}
-  <div class="p-strip--light is-shallow distro-background">
-    <style>
-      .distro-background {
-        background-image: linear-gradient(-90deg, {{ distro_color_1 }}, {{ distro_color_2 }});
-      }
-    </style>
-    <div class="row">
-      <div class="col-8">
-        <h1 class="p-heading--two" style="color: #f7f7f7">How to install {{ snap_title }}<br/> on <b>{{ distro_name }}</b></h1>
+  <section class="p-strip--light distro-banner u-no-padding--top">
+    <div class="distro-banner__background">
+      <svg class="distro-banner__suru" viewBox="0 0 1440 776" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <defs>
+          <linearGradient x1="100%" y1="50%" x2="0%" y2="50%" id="b">
+            <stop stop-color="{{ distro_color_1 }}" offset="0%"/>
+            <stop stop-color="{{ distro_color_2 }}" offset="100%"/>
+          </linearGradient>
+          <path id="a" d="M0 0h1440v776H0z"/>
+          <linearGradient x1="100%" y1="50%" x2="0%" y2="50%" id="c">
+            <stop stop-color="{{ distro_color_2 }}" offset="0%"/>
+            <stop stop-color="{{ distro_color_1 }}" offset="100%"/>
+          </linearGradient>
+        </defs>
+        <g fill="none" fill-rule="evenodd">
+          <mask id="d" fill="#fff"><use xlink:href="#a"/></mask>
+          <use fill="url(#b)" xlink:href="#a"/>
+          <path fill="url(#c)" mask="url(#d)" d="M0 169l295 607H0V169L1440 0v776H703l737-432V0z"/>
+        </g>
+      </svg>
+      <div class="row">
+        <div class="mobile-col-2 col-8"></div>
+        <div class="mobile-col-2 col-4">
+          <img src="{{ distro_logo }}" />
+        </div>
       </div>
-      <div class="col-3">
-        <img src="{{ distro_logo }}" />
+    </div>
+    <div class="p-strip">
+      <div class="row">
+        <h1 class="p-heading--two u-no-margin">How to install {{ snap_title }}<br/> on <b>{{ distro_name }}</b></h1>
       </div>
     </div>
-  </div>
-  <div class="p-strip is-shallow">
+
     <div class="row">
-      <div class="p-snap-heading">
-        {% if icon_url %}
-          <img class="p-snap-heading__icon" src="{{ icon_url }}" alt="{{ snap_title }} snap" data-live="icon" />
-        {% else %}
-          <img class="p-snap-heading__icon" src="https://assets.ubuntu.com/v1/be6eb412-snapcraft-missing-icon.svg" alt="" data-live="icon" />
-        {% endif %}
-        <div class="p-snap-heading__title">
-          <h1 class="p-heading--two p-snap-heading__name" data-live="title">{{ snap_title }}</h1>
-          <div class="u-hide u-show--small">
-            {% include 'store/_snap-header-information.html' %}
+      <div class="p-card">
+        <div class="row">
+          <div class="details-block">
+            <div class="p-snap-heading">
+              {% if icon_url %}
+                <img class="p-snap-heading__icon" src="{{ icon_url }}" alt="{{ snap_title }} snap" data-live="icon" />
```

Comment:
should probably remove the `data-live` attributes

Reply:
Ah, forgot about those.

Reply:
@Lukewh on a second thought... They don't really hurt here and I plan to actually move these common parts to some partials to reuse them in both snap details and here to keep them consistent later. So the `data-live` would need to be shared in both places as well (unless we add them only based on some parameter).